app/models/blueprint.rb

```diff
@@ -49,6 +49,15 @@ def update_bundle(options)
     # TODO
   end
 
+  def serialized_bundle
+    result = bundle.as_json.dup
+    result["service_templates"] = bundle.descendants.map(&:as_json)
+    result["service_catalog"] = bundle.service_template_catalog.as_json
+    result["service_dialog"] = bundle.dialogs.first.as_json
```

Comment:
@abellotti We probably shouldn't rely on this being so, I think we can get it through the resource_actions as you say